druid-shell/src/dialog.rs

```diff
@@ -85,6 +85,15 @@ impl FileDialogOptions {
         self.show_hidden = true;
         self
     }
+    pub fn select_directories(mut self) -> Self {
```

Comment:
before we merge I'd like docs for the public methods.

Reply:
Should I also add tests? Although I'm not really sure how I'd test this, given basically all functionality is being provided by the underlying window manager...

Reply:
yea, I wouldn't worry about tests.